src/Traits/GetDependenciesTrait.php

```diff
@@ -11,10 +11,10 @@
 
 trait GetDependenciesTrait
 {
-    protected function resolveClassMethodDependencies(array $parameters, $instance, $method)
+    protected function resolveClassMethodDependencies($instance, $method)
     {
         if (!method_exists($instance, $method)) {
-            return $parameters;
+            return [];
```

Comment:
I returned an empty array instead of a `null` value regarding the capability with the only usage on the `SwaggerService` on line 656.

https://github.com/RonasIT/laravel-swagger/blob/07aab91443233d767184b7f2d73f60610380e4bc/src/Services/SwaggerService.php#L658-L660

A `null` value needs to be handled as the first parameter for `Arr::first()`.